src/Lumi/Components2/Image.purs

```diff
@@ -0,0 +1,98 @@
+module Lumi.Components2.Image
+  ( imageThumb
+  , small
+  , medium
+  , large
+  , extraLarge
+  , _customSize
+  , round
+  ) where
+
+import Prelude
+
+import Data.Maybe (Maybe(..), fromMaybe)
+import Data.Monoid as Monoid
+import Data.Nullable as Nullable
+import Data.String as String
+import Data.Tuple.Nested ((/\))
+import Effect.Unsafe (unsafePerformEffect)
+import Lumi.Components (LumiComponent, PropsModifier, lumiComponent, ($$$))
+import Lumi.Components.Loader (loader)
+import Lumi.Components.Spacing (Space)
+import Lumi.Components.Svg (userSvg)
+import Lumi.Components2.Box as Box
+import Lumi.Styles (style, toCSS)
+import Lumi.Styles.Box (FlexAlign(..))
+import Lumi.Styles.Box as Styles.Box
+import Lumi.Styles.Image as Styles.Image
+import Lumi.Styles.Slat hiding (_interactive,slat) as Styles.Slat
+import Lumi.Styles.Theme (LumiTheme(..), useTheme)
+import React.Basic.DOM as R
+import React.Basic.Emotion as E
+import React.Basic.Events (handler_)
+import React.Basic.Hooks (JSX)
+import React.Basic.Hooks as Hooks
+
+type ImageProps
+  = ( src :: String
+    , placeholder :: Maybe JSX
+    )
+
+imageThumb :: LumiComponent ImageProps
+imageThumb =
```

Comment:
Should we call this component `thumbnail` instead if it's only for thumbnail images?

Reply:
I think the loader functionality might be useful for more than thumbnail images. Maybe we can define an `image` component with no size restrictions and then build `thumbnail` off of that, kind of like `paragraph` builds off of `text`?

Reply:
Ah that's a good idea!

Reply:
Oh nice, yeah I'll break out an generic `image` component on which we can build the `thumbnail`.